addons/mail/static/tests/tours/discuss_sub_channel_search_tour.js

```diff
@@ -72,56 +69,57 @@ registry.category("web_tour.tours").add("test_discuss_sub_channel_search", {
         {
             trigger:
                 ".o-mail-SubChannelList .o-mail-SubChannelPreview:count(31):contains(Sub Channel 99)",
-            async run() {
+            async run({ waitFor }) {
                 // Already fetched sub channels are shown in addition to the one
                 // that was fetched during the search.
                 for (let i = 99; i > 69; i--) {
-                    await contains(".o-mail-SubChannelPreview", {
-                        text: `Sub Channel ${i}`,
-                    });
+                    await waitFor(`.o-mail-SubChannelPreview:contains(Sub Channel ${i})`);
                 }
-                await contains(".o-mail-SubChannelPreview", { text: `Sub Channel 10` });
+                await waitFor(".o-mail-SubChannelPreview:contains(Sub Channel 10)");
                 // Ensure lazy loading is still working after a search.
                 await waitForLoadMoreToDisappearDef;
                 waitForLoadMoreToDisappearDef = new Deferred();
-                await scroll(".o-mail-ActionPanel:has(.o-mail-SubChannelList)", "bottom");
             },
         },
+        {
+            trigger: ".o-mail-ActionPanel:has(.o-mail-SubChannelList)",
+            run: "scroll bottom",
+        },
         {
             trigger:
                 ".o-mail-SubChannelList .o-mail-SubChannelPreview:count(61):contains(Sub Channel 40)",
-            async run() {
+            async run({ waitFor }) {
                 for (let i = 99; i > 39; i--) {
-                    await contains(".o-mail-SubChannelPreview", {
-                        text: `Sub Channel ${i}`,
-                    });
+                    await waitFor(`.o-mail-SubChannelPreview:contains(Sub Channel ${i})`);
```

Comment:
Please use `:text()` instead as we want to keep exact match. Same remark in the whole diff.